x/blob/keeper/keeper.go

```diff
@@ -61,9 +61,13 @@ func (k Keeper) PayForBlob(goCtx context.Context, msg *types.MsgPayForBlob) (*ty
 	gasToConsume := uint64(shares.MsgSharesUsed(int(msg.BlobSize)) * GasPerMsgShare)
 	ctx.GasMeter().ConsumeGas(gasToConsume, payForBlobGasDescriptor)
 
-	ctx.EventManager().EmitEvent(
-		types.NewPayForBlobEvent(sdk.AccAddress(msg.Signer).String(), msg.GetBlobSize()),
-	)
```

Comment:
Can remove this (`types.NewPayForBlobEvent`) method as it's not being used, otherwise can delegate
```go
&types.EventPayForBlob{
	Signer:   sdk.AccAddress(msg.Signer).String(),
	BlobSize: msg.GetBlobSize(),
}
```
to above mentioned method and change the signature.